babel.config.js

```diff
@@ -50,6 +51,17 @@ module.exports = function(api) {
       break;
   }
 
+  if (includeRuntime) {
+    const babelRuntimePackageJSONPath = require.resolve(
+      "@babel/runtime/package.json"
+    );
+    const path = require("path");
```

Comment:
It's a simple question. What is the advantage of declaring within this scope rather than at the top?

Reply:
Not a strong reason: `require("path")` is used only when building babel-*-standalone. However, this config is shared with other building process (build/jest), by doing so we could avoid the overhead of requiring a builtin module.